cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -335,6 +337,8 @@ var createModeSubcommands = map[string]modeSubcommand{
 
 			subcmd.Flags().BoolVar(&o.IgnoreUnknown, ignoreUnknownFlagName, false,
 				"Ignore all the unsupported flags in the bash script.")
+			subcmd.Flags().StringVar(&o.InitImage, initImageFlagName, "",
```

Comment:
```suggestion
			subcmd.Flags().StringVar(&o.InitImage, initImageFlagName, slurmDefaultInitImage,
```

Shouldn't we set the default image to the flag, right?

Reply:
Done